CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing
+
+This doc explains how to get started contributing to [baikonur-oss](https://github.com/baikonur-oss) project.
+
+## Filing an Issue
+
+If you are trying to use baikonur-oss modules and run into some issues, please file an issue. We can look into the issue more easily if you describe in details as possible as you can. The following is example detailed descriptions:
+
+- Version of tools and modules
+- Expected and actual results
+- Error messages with debug option
+- Steps to reporoduce the issues
+
+## Submitting a PR
+
+Before submitting a PR, make sure that there's an issue filed for your work. It might need some discussion before addressing the issue. Filing a issue first will help us to save your time to solving misunderstandings between you and us.
```

Comment:
I doubt we need this section. 

Instead, I think we should say something like this:
```
PRs are welcome! Submitting an issue and discussing is a good practice, so feel free to open new issues. Baikonur project is young, and we still have to figure out guidelines etc., so submitting an issue prior to submitting a PR might be better.
```